query/src/api/rpc/exchange/exchange_manager.rs

```diff
@@ -80,24 +71,61 @@ impl DataExchangeManager {
 
     // Create connections for cluster all nodes. We will push data through this connection.
     pub async fn init_nodes_channel(&self, packet: &InitNodesChannelPacket) -> Result<()> {
-        let mut exchange_senders = HashMap::with_capacity(packet.target_2_fragments.len());
-        for target in packet.target_2_fragments.keys() {
-            if target != &packet.executor {
-                let config = self.config.clone();
-                let exchange_sender = ExchangeSender::create(config, packet, target).await?;
-                exchange_senders.insert(target.clone(), exchange_sender);
+        let mut request_exchanges = vec![];
+        let mut targets_exchanges = HashMap::new();
+
+        let source = &packet.executor.id;
+        for connection_info in &packet.connections_info {
+            if connection_info.create_request_channel {
+                let query_id = &packet.query_id;
+                let address = &connection_info.target.flight_address;
+                let mut flight_client = Self::create_client(&self.config, address).await?;
```

Comment:
What's the client timeout number?

Reply:
We use the default connection timeout and the fixed sending and receiving timeout (60 seconds). I have created a new issue to track this. #7092